### DISCUSSION TITLE
Option to preserve lower case first letter in custom titles

### DISCUSSION THREAD

**Discussion by zacyu** [JulianPrieber/littlelink-custom#733] [Feature requests]
Custom titles are stored as case sensitive strings. However, when rendered on user pages, the first letter of the custom title is always capitalized (e.g., rendering "macOS" as "MacOS"). How do you feel about implementing an option (e.g., in appearance) to opt out of this behavior?

**Comment by JulianPrieber** [comment]
![chrome_lwKq1Sb0Ml](https://github.com/LinkStackOrg/LinkStack/assets/60265788/117cb4b9-0d74-435c-8e5f-0840df50042d)
Custom buttons can do this already, but as of [v4.7.0](https://github.com/LinkStackOrg/LinkStack/releases/tag/v4.7.0) this can be considered a bug. I'll look into it.

**Reply by zacyu** [reply]
Thanks for looking into it and identifying the workaround! Confirming that custom button texts are not impacted.
